src/pages/publication/PublicationFormTabs.tsx

```diff
@@ -42,7 +42,7 @@ export const PublicationFormTabs: React.FC<PublicationFormTabsProps> = ({
       <LinkTab
         label={`2. ${t('heading.description')}`}
         {...a11yProps('description')}
-        error={hasTouchedError(errors.description, touched.description)}
+        // error={hasTouchedError(errors.description, touched.description)}
```

Comment:
Dropper å vise om en tab er OK eller ikke... tar det som egen PR

Reply:
da lager du egen task i Jira også ? ;-)  (og for todos)